includes/Core/Assets/Assets.php

```diff
@@ -347,7 +347,7 @@ private function get_assets() {
 		$dependencies = array(
 			'googlesitekit-vendor',
 			'googlesitekit-commons',
-			'googlesitekit-base',
+			'googlesitekit-admin',
```

Comment:
This shouldn't be changed to `admin`. Rather, let's rename the actual file `googlesitekit-admin.js` to `googlesitekit-base.js` which was the original plan here.

Reply:
Updated.